connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/connect/client/SparkResult.scala

```diff
@@ -45,7 +45,7 @@ private[sql] class SparkResult[T](
   private[this] var numRecords: Int = 0
   private[this] var structType: StructType = _
   private[this] var boundEncoder: ExpressionEncoder[T] = _
-  private[this] val batches = mutable.Buffer.empty[ColumnarBatch]
+  private[sql] val batches = mutable.Buffer.empty[ColumnarBatch]
```

Comment:
Please expose the metric you need for testing instead of making mutable state more visible.

Reply:
Sure, changed back to private.